packages/fluentui/react-icons-northstar/src/index.ts

```diff
@@ -86,6 +87,11 @@ export { default as FilesUploadIcon } from './components/FilesUploadIcon';
 export { default as FilesVideoIcon } from './components/FilesVideoIcon';
 export { default as FilesZipIcon } from './components/FilesZipIcon';
 export { default as FlagIcon } from './components/FlagIcon';
+export { default as FluentFileGenericIcon } from './components/FluentFileGenericIcon';
```

Comment:
Why did we start creating icons with the fluent prefix?

Reply:
Because they come from Fluent repo. It was requested that they be added to Webclient, so this means adding here too. Kept the prefix to disticguish them. IS there other way?

Reply:
Just worried that if we start like this, we will end up with having half the icons with Fluent prefix.. If there is a good enough reason and we are tracking where the icons comes from then ok..

Reply:
I'll ask around a little.

Reply:
I agree with @mnajdova. The icons might come from Fluent, but I don't see a reason to make that distinction here. Here we just have icons that Teams uses, unless there is some other distinction I'm missing...